core/src/scene/scene.h

```diff
@@ -103,6 +105,9 @@ class Scene {
     const auto& featureSelection() const { return m_featureSelection; }
     const auto& sceneAssets() const { return m_sceneAssets; };
 
+    void createSceneAsset(const std::shared_ptr<Platform>& platform, const Url& resolvedUrl, const Url& relativeUrl,
+                          const Url& base);
+
```

Comment:
No need for this PR - I would rename these to`createAsset() / assets()` as it's clear that it's about parts of the scene